release_notes/Service_Fabric_ReleaseNotes_90CU2.md

```diff
@@ -0,0 +1,95 @@
+* .NET 5.0 runtime is reaching end-of-life on May 8, 2022.Service Fabric releases after that date will drop support for Service Fabric applications running with .NET 5.0 runtime.Current applications running on .NET 5.0 runtime will continue to work, but requests for investigations or request for changes will no longer be entertained. Please migrate to using .NET 6.0 version instead.
+* Ubuntu 16.04 LTS reached its 5-year end-of-life window on April 30, 2021.Service Fabric runtime has dropped support for 16.04 LTS, and we recommend moving your clusters and applications to Ubuntu 18.04 LTS or 20.04 LTS.Current applications running on it will continue to work, but requests for investigations or requests for change will no longer be entertained.This would be the last release containing packages for Ubuntu 16.04 LTS. Please migrate to Ubuntu 18.04 or 20.04 instead.
+* Service Fabric runtime will soon be stop using BinaryFormatter based remoting exception serialization by default and move to using Data Contract Serilization (DCS) based remoting exception serialization by default.Current applications using it will continue to work as-is, but Service Fabric strongly recommends moving to using Data Contract Serilization (DCS) based remoting exception instead.
+* Service Fabric runtime will soon be archiving and removing Service Fabric runtime version 6.4 packages and older, as well as SDK version 3.3 packages and older from the package Download Center.
+
+
+## Service Fabric Common Bug Fixes
+
+| Versions | IssueType | Description | Resolution | 
+|-|-|-|-|
+| **Windows - 9.0.1048.9590<br>Ubuntu 16 - 9.0.1056.1<br>Ubuntu 18 - 9.0.1056.1** | **Bug** | Reverse Proxy | **Brief desc**: Sending long URI request over Reverse Proxy causes an error.<br>**Impact**: Reverse Proxy (FabricApplicationGateway) would incorrectly return FABRIC_E_INVALID_ADDRESS when the path component in a URI request was over internal max limit<br>**Fix**: Support long path request URI over Reverse Proxy.
+| **Windows - 9.0.1048.9590<br>Ubuntu 16 - 9.0.1056.1<br>Ubuntu 18 - 9.0.1056.1** | **Bug** | Failover Manager cache | **Brief desc**: Cache cleanup logic of Failover Manager had a bug which resulted in memory increase and performance degradation.The longer the cluster ran, the greater memory footprint would be, and performance would incrementally decrease.<br>**Impact**: This can cause clusters to be extremely unresponsive and slow.<br>**Fix**: Add cleanup logic that purges all stale entries keeping the load cache small and predictable. 
+| **Windows - 9.0.1048.9590<br>Ubuntu 16 - 9.0.1056.1<br>Ubuntu 18 - 9.0.1056.1** | **Bug** | Cluster Manager Service | **Brief desc**: Service Fabric upgrades gets stuck in an upgrade domain.<br>**Impact**: Cluster Manager does not persist the current Service Fabric upgrade after the Cluster Manager primary is failed over causing stuck upgrades in a Upgrade Domain.<br>**Fix**: Cluster Manager will retry recovered operations after failover. 
```

Comment:
Should we link to the TSG if a customer is affected?

Reply:
There is no TSG, the communication was done through mail